deployment/nixops/prometheus.nix

```diff
@@ -180,5 +180,43 @@ in
         ];
       };
     };
+
+    systemd.services.githubhooks = {
+      enable = enableGithubHooks;
+      path = ["${githubhooks}" pkgs.git pkgs.nixops pkgs.nix pkgs.gnutar pkgs.gzip ];
+      script = "deployment-server-exe --slackChannel tmp_meadowinthecloud --keyfile ${configDir}/secrets.json --port 8080 --configDir ${configDir} --stateFile /root/.nixops/deployments.nixops --include nixos=/root/.nix-defexpr/channels/nixos --include nixpkgs=${nixpkgsLocation}";
```

Comment:
minor but I find the mix of calling it `githubhooks` vs `deployment-server-exe` confusing.

Reply:
Make slack channel a variable?

Reply:
Should we be using root's state file? Maybe it should be a specific different one?

Reply:
I don't think there is a way around running this service as root so which state file we use is not that relevant I don't think. Also this makes manually working with nixops easier as you ssh in as root and nixops commands just work.

Slack channel can be a variable, is it worth it? When will we change it?

Do you think I should change `githubhooks` to `deployment-server` or the other way round? I take your point.

Reply:
I just get worried when I see constants. Maybe it's just me but I like having even single-use variables that show that I thought about it and yes it represents this thing and is supposed to have this value.

Reply:
Ok

Reply:
renamed to `deploymentServer`